CHANGELOG.md

```diff
@@ -1,5 +1,50 @@
 # [Versions](https://mui.com/versions/)
 
+## 6.1.10
+
+<!-- generated comparing v6.1.9..master -->
+
+_Dec 3, 2024_
+
+A big thanks to the 10 contributors who made this release possible.
+
+### `@mui/material@6.1.10`
+
+- [Avatar] Fix `slotProps.img` not spread to hook (#44536) @siriwatknp
```

Comment:
Not related to this PR, but we should give better titles on the PRs. For example this one describes the fix, not the bug that was solved. I am mentioning because for someone reading the changelog, reading which bug was fixed is much more important than how we fixed it.

Reply:
Totally agree